GVFS/GVFS.Common/Git/GitProcess.cs

```diff
@@ -529,7 +529,7 @@ public Result PackObjects(string filenamePrefix, string gitObjectsDirectory, Act
         /// </summary>
         public Result WriteCommitGraph(string objectDir, List<string> packs)
         {
-            string command = "commit-graph write --stdin-packs --append --object-dir \"" + objectDir + "\"";
+            string command = "commit-graph write --stdin-packs --split --size-multiple=4 --object-dir \"" + objectDir + "\"";

```

Comment:
Do we need some tests the write the old and convert to incremental and the other way as well or I'm guessing there are git tests for that?

Reply:
`t/t5323-split-commit-graph.sh` has [a test that upgrades from the "flat" file to a chain](https://github.com/microsoft/git/blob/vfs-2.22.0/t/t5324-split-commit-graph.sh#L108-L120).

Reply:
Also is it git that is moving the file from the `info/commit-graph` location to the new `info/commit-graphs/X` with the `split` command?

Reply:
Yes. Git realizes that we are adding commits on top of an existing `info/commit-graph` file and moves it to `info/commit-graphs/graph-{hash}.graph` before completing the write of the next level (the data is already written to a temp file before the move) and updating the `info/commit-graphs/commit-graph-chain` file to point to it.

During this "upgrade" step, there is a brief moment where `info/commit-graph` does not exist and `info/commit-graphs/commit-graph-chain` does not exist yet. A process that looks for a commit-graph in that window may have slow performance, but not incorrect results.